razzle.config.js

```diff
@@ -54,6 +57,23 @@ module.exports = {
       };
     }
 
+    if (target === 'web') {
+      appConfig.plugins.push(
+        new OfflinePlugin({
+          appShell: '/article/',
+          autoUpdate: 1000 * 60 * 2,
```

Comment:
Setting this to 2 minutes seems awfully low, unless you are considering PWA still under development and want faster updates.

Also, please add a comment that this is in milliseconds.

Reply:
Also look into divergent caching strategies for different assets, is it possible and is there an easy way to do it?